lib/ansible/modules/network/checkpoint/checkpoint_access_layer_facts.py

```diff
@@ -52,10 +52,10 @@
 """
 
 RETURN = """
-ansible_facts:
-  description: The checkpoint access layer facts.
+api_result:
```

Comment:
@justjais because I wanted it to be consistent, means all the files return api_result.. (all the modules and all the commands like install policy return api_result).
Isn't it better?

Reply:
@chkp-orso to be consistent with other Ansible resource module, it's better to have as is. Also, it's better to have each module returns the module facts instead of common return.

Reply:
@justjais So to write "ansible_facts" or "checkpoint_access_layer_facts" ?

Reply:
@chkp-orso for already existing module, u need not to change the return values, respective comment was for future modules

Reply:
@justjais amm but I would prefer all the modules to be consistent..
because I create all the modules automatically with the tool..

Reply:
So I would prefer to return "ansible_facts" or "checkpoint_access_layer_facts" to all files

Reply:
@chkp-orso It's better to return the facts with the module name, but if not u can return with `ansible_facts` for e.g. for module `checkpoint_host` the return value is `checkpoint_hosts` but for module `checkpoint_access_layer_facts` the return value is `ansible_facts`

Reply:
@justjais OK, so anyway because we prefer to be consistent, we will return "ansible_facts" to all the modules